shimmingtoolbox/dicom_to_nifti.py

```diff
@@ -0,0 +1,134 @@
+import os
+import numpy as np
+from .read_nii import read_nii
+
+
+def dicom_to_nifti(unsortedDicomDir, niftiPath):
+    """
+    TODO: indicate the purpose of this function
+    :param unsortedDicomDir:
+    :param niftiPath:
+    :return:
+    """
+
+    #os.mkdir(niftiPath)
```

Comment:
You're going to want `os.makedirs` here. It's like `mkdir -p`: it works recursively.